src/Azure/Azure.Quantum.Client.Test/Helpers/TestConstants.cs

```diff
@@ -5,10 +5,11 @@ namespace Microsoft.Azure.Quantum.Test
 {
     public static class TestConstants
     {
-        public const string SubscriptionId = "sub1";
-        public const string ResourceGroupName = "rg1";
-        public const string WorkspaceName = "ws1";
-        public const string ProviderId = "provider1";
-        public const string Endpoint = "https://test";
+        // Used when connecting live
+        public const string LiveSubscriptionId = "916dfd6d-030c-4bd9-b579-7bb6d1926e97";
```

Comment:
Shouldn't this values live in Environment Variables instead of the code?

Reply:
I have changed the tests to give preference to the same env variables we use for e2e.